phalcon/mvc/model/query/builder.zep

```diff
@@ -401,7 +401,6 @@ class Builder implements BuilderInterface, InjectionAwareInterface
 	 *	$builder->innerJoin('Robots');
 	 *	$builder->innerJoin('Robots', 'r.id = RobotsParts.robots_id');
 	 *	$builder->innerJoin('Robots', 'r.id = RobotsParts.robots_id', 'r');
-	 *	$builder->innerJoin('Robots', 'r.id = RobotsParts.robots_id', 'r', 'LEFT');
```

Comment:
method name is "innerJoin", not type parameter:
https://github.com/phalcon/cphalcon/pull/9923/files#diff-6f4bf16701c35223bb36210003676fdfL413